libmscore/stafftype.cpp

```diff
@@ -241,6 +243,7 @@ void StaffType::write(Xml& xml) const
             xml.tag("stemsDown",        _stemsDown);
             xml.tag("stemsThrough",     _stemsThrough);
             xml.tag("upsideDown",       _upsideDown);
+            xml.tag("showTabFingering", _showTabFingering);
```

Comment:
In master (at least) @lasconic  liked another method better:
`xml.tag("showTabFingering", _showTabFingering, false);`